server/models/Notifications.js

```diff
@@ -0,0 +1,24 @@
+const mongoose = require("mongoose");
+
+const notificationsSchema = new mongoose.Schema({
+  notifications: [
+    {
+      isNewNotification: {
+        type: Boolean,
+        default: true,
+      },
+      title: String,
+      body: String,
+      date: Date,
```

Comment:
This would always be the date it's created right? You can pass the [timestamps option](https://mongoosejs.com/docs/guide.html#timestamps) to the Schema declaration to handle this

Reply:
Oh true, that would simplify things a bit, thanks.